src/pages/Home/HomePage.js

```diff
@@ -1,9 +1,60 @@
 import { FeaturedCarousel } from "./components/FeaturedCarousel"
+import Book4 from "../../assets/book4.jpg";
+import Book5 from "../../assets/book5.jpg";
+import Book6 from "../../assets/book6.jpg";
+import Book7 from "../../assets/book7.jpg";
+import Book8 from "../../assets/book8.jpg";
+import Book9 from "../../assets/book9.jpg";
+// Array of book details
+const books = [
+  {
+    id: 1,
+    imageUrl: Book4, 
+  },
+  {
+    id: 2,
+    imageUrl: Book5, 
+  },
+  {
+    id: 3,
+    imageUrl: Book6, 
+  },
+  {
+    id: 4,
+    imageUrl: Book7, 
+  },
+  {
+    id: 5,
+    imageUrl: Book8, 
+  },
+  {
+    id: 6,
+    imageUrl: Book9, 
+  },
+];
+
+// Book component
+const Book = ({ title, imageUrl }) => (
```

Comment:
Maybe extract this into a new component file (src/components). Since this card will be used to display a genre or chapter as well, maybe rename this to CarouselItem or something more generic